modules/scripts/startup-script/examples/install_ansible.sh

```diff
@@ -185,8 +190,9 @@ main() {
 	fi
 	if [ -z "${ansible_version}" ] || [ "${ansible_major_vers}" -ne "${ansible_req_major_vers}" ] ||
 		[ "${ansible_minor_vers}" -lt "${ansible_req_minor_vers}" ]; then
-		${python_path} -m pip install ansible==${REQ_ANSIBLE_PIP_VERSION}
+		${python_path} -m pip install ansible-core==${REQ_ANSIBLE_VERSION}
 	fi
+	${python_path} -m pip cache purge
```

Comment:
It is annoying for user applications to have a pip cache with packages that (by the time they run `pip install`) may be out of date relative to the ones they want.